tmt/base.py

```diff
@@ -1578,19 +1656,19 @@ def import_plan(self) -> Optional['Plan']:
             if self.my_run and not self.my_run.opt('dry'):
                 assert self.parent is not None  # narrow type
                 assert self.parent.workdir is not None  # narrow type
-                destination = os.path.join(self.parent.workdir, "import", self.name.lstrip("/"))
+                destination = self.parent.workdir / "import" / self.name.lstrip("/")
                 if plan_id.url is None:
                     raise tmt.utils.SpecificationError(
                         f"No url provided for remote plan '{self.name}'.")
-                if os.path.exists(destination):
+                if destination.exists():
                     self.debug(f"Seems that '{destination}' has been already cloned.", level=3)
                 else:
                     tmt.utils.git_clone(plan_id.url, destination, self)
                 if plan_id.ref:
                     self.run(Command('git', 'checkout', plan_id.ref), cwd=destination)
                 if plan_id.path:
-                    destination = os.path.join(destination, plan_id.path.lstrip("/"))
-                node = fmf.Tree(destination).find(plan_id.name)
+                    destination = destination / plan_id.path.relative_to('/')
```

Comment:
The `plan_id.path` does not have to start with a slash. Should be fixed by 40675b5c.

Reply:
Test passed, looks to be ok.